src/main/java/qna/domain/Answers.java

```diff
@@ -0,0 +1,26 @@
+package qna.domain;
+
+import qna.CannotDeleteException;
+
+import java.util.List;
+
+public class Answers {
+
+    List<Answer> answers;
```

Comment:
다음과 같이 변경된다면 어떤 장점이 있을까요 :) ?
```suggestion
    private final List<Answer> answers;
```

Reply:
해당 변수에 Private final를 사용하면 내부에서만 해당 변수의 접근이 가능하고
외부에서 해당변수를 재 초기화하지 못하도록 할수 있을거 같습니다!